Cargo.toml

```diff
@@ -20,6 +20,10 @@ members = [
     "relm4-macros",
 ]
 
+[package.metadata.docs.rs]
+features = ["dox"]
```

Comment:
I wonder whether it makes sense to use `all-features = true`. Otherwise, the code that is behind the "libadwaita" and "libpanel" feature flags won't be part of the documentation.

Reply:
Good idea